apps/docs/content/guides/platform/org-based-billing.mdx

```diff
@@ -33,6 +33,8 @@ Quotas and features are applied to your entire organization, independent of how
 
 Upgrading your organization to a paid plan means that you unlock the benefits of the paid plan for **all** projects within that organization - for example, no projects in your "Pro" organization will be paused. See [pricing page](https://supabase.com/pricing) for details on all plans.
 
+It is not possible to mix different plans inside a single organizations. You cannot have one Pro plan project and one Free plan project inside a single organization. To have projects on different plans, you'll need to create different organizations. Also see [Project Transfers](/docs/guides/platform/project-transfer) in case you're looking to transfer a project to a different organization.
```

Comment:
```suggestion
It is not possible to mix different plans inside a single organization. You cannot have one Pro plan project and one Free plan project inside a single organization. To have projects on different plans, you'll need to create different organizations. Also see [Project Transfers](/docs/guides/platform/project-transfer) in case you're looking to transfer a project to a different organization.
```